src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -39,4 +40,26 @@ public List<AdvertBasicInfoDTO> getByCategory(@PathVariable String category) {
         return advertService.getByCategory(category);
 
     }
+
+    @DeleteMapping("/advert/{id}")
+    public void deleteAdvert(@PathVariable Long id) {
+        advertService.deleteAdvertById(id);
+    }
+
+    @DeleteMapping("/advert/title/{title}")
```

Comment:
`/title/{title}` here and the same pattern for the others

Reply:
changes have been made